hail/src/main/scala/is/hail/expr/types/encoded/EType.scala

```diff
@@ -157,46 +159,84 @@ trait DecoderAsmFunction { def apply(r: Region, in: InputBuffer): Long }
 trait EncoderAsmFunction { def apply(off: Long, out: OutputBuffer): Unit }
 
 object EType {
+
+  val cacheCapacity = 256
```

Comment:
I'm not sure this is big enough.

Reply:
why not?

Reply:
I mean, why invalidate at all? JVM classes are teeny compared to other areas of pressure on memory.

Reply:
if we put this cache on the ExecuteContext, we could have it unbounded. But Hail JVMs are sometimes very long-running, executing many pipelines. That would cause this cache to grow without bound, which isn't great.